### PR TITLE
B&R guide restructuring: tidying up

### PR DESCRIPTION
**Pull request by catong**
This PR tidies up loose ends from restructuring including an attempt to make top level headings consistent in using gerunds. 
@lamagnifica Please take a look. Thanks!

### CODE REVIEW

File: en_us/course_authors/source/video/index.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .. _Processing Video Files Index:
 
 ############################
-Processing Video Files
```
Comment (catong) [inline en_us/course_authors/source/video/index.rst:4]: @lamagnifica Made this change per the restructuring proposal wiki; in line with the sections above and below, which are currently "Adding Course Components" and  "Creating Exercises and Tools". Do you prefer "Creating Video Content" or "Adding Video Content"?

Comment (lamagnifica) [inline en_us/course_authors/source/video/index.rst:4]: Adding please

File: en_us/course_authors/source/course_features/index.rst [hunk 2]
```diff
@@ -1,10 +1,11 @@
 .. _Course Features Index:
 
-#################################
-Choosing Features For Your Course
-#################################
+###############################################
+Choosing Enhanced Capabilities For Your Course
+###############################################
```
Comment (lamagnifica) [inline en_us/course_authors/source/course_features/index.rst:5]: oooh I like it!

**Comment by lamagnifica** [comment]
:+1: outstanding!

**Comment by catong** [comment]
Squashed commits; will merge after all tests pass.

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/701

**Pull request merged**